src/prplmesh_getall.lua

```diff
@@ -68,4 +68,184 @@
 #
 ################################################################################
 --]]
-print("0; 1;")
+
+require("ubus")
+
+function path(str)
+    local res = str:gsub("(.*)%s.*$","%1")
+    print(res)
+    return res
+end
+
+
+function get_indexes(path)
+
+    local _ubus_connection
+    local devices
+    local instances
+    local result = {}
+
+    _ubus_connection = ubus.connect()
+    if not _ubus_connection then
+        print("Ubus sock conn fail")
+        return false
+    end
+
+    -- Retrieve output from list method via path from UBus
+    instance = _ubus_connection:call(path, "list", { })
+    if not instance then
+        print(path .. " not found in UBus")
+        return false
+    end
+
+    -- Fill up path instance indexes in table.
+    for k, v in pairs(instance["instances"]) do
+        result[k] = v["index"]
+        --print(" value: " .. tostring(result[k]))
+    end
+
+    _ubus_connection:close()
```

Comment:
You should close ubus connection just after receiving alone result which you need
Because you have a `return` statement above this line and may receive non closed connection

Reply:
fixed